Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -344,6 +374,17 @@ class Bricks final {
         m_state = GameState::Active;
     }
 
+    void toggle_pause()
+    {
+        if (m_state == GameState::GameOver) {
+            return;
+        } else if (m_state == GameState::Active) {
+            m_state = GameState::Paused;
+        } else {
+            m_state = GameState::Active;
+        }
```

Comment:
One-line `if` blocks do not need braces; i.e.:
```suggestion
        if (m_state == GameState::GameOver)
            return;
        else if (m_state == GameState::Active)
            m_state = GameState::Paused;
        else
            m_state = GameState::Active;
```

Reply:
Sorry its a habit I've picked up from a C project I was working on. Thanks for the help.

Reply:
Also there's no need for an `else` after a branch that returns.  So you could do:
```c++
        if (m_state == GameState::GameOver)
            return;
        m_state = m_state == GameState::Active ? GameState::Paused : GameState::Active;
```